config/locales/fr.yml

```diff
@@ -224,7 +224,15 @@ fr:
       other:
   back:
   cancel: Annuler
-  cancel_file_transfer:
+  cancel_file_transfer: 
+  certificate:
```

Comment:
@raakal @y2749zha Before this can be deployed to the production site we will have to translate the certificate text